modules/programs/gh.nix

```
@@ -106,5 +115,12 @@ in {
     programs.git.extraConfig.credential."https://github.com".helper =
       mkIf cfg.enableGitCredentialHelper
       "${cfg.package}/bin/gh auth git-credential";
+
+    xdg.dataFile."gh/extensions" = mkIf (cfg.extensions != null) {
+      source = toString (pkgs.linkFarm "gh-extensions" (builtins.map (p: {
```

I don't think you need `toString` here
```suggestion
      source = pkgs.linkFarm "gh-extensions" (builtins.map (p: {
```